### ISSUE TITLE
falcon_kit.mains.ovlp_to_graph fails when using the most recent networkx 2.4 library

### ISSUE THREAD

**Issue by ibebio** [PacificBiosciences/pbbioconda#230]
Hi,
I discovered a possible bug in lib/python3.7/site-packages/falcon_kit/mains/ovlp_to_graph.py when a fresh install from bioconda is created.
```
# Given preads.ovl,
# write sg_edges_list, c_path, utg_data, ctg_paths.
time python3 -m falcon_kit.mains.ovlp_to_graph  --min-len 10000 --overlap-file preads.ovl >| fc_ovlp_to_graph.log
+ python3 -m falcon_kit.mains.ovlp_to_graph --min-len 10000 --overlap-file preads.ovl
falcon-kit 1.4.4
Traceback (most recent call last):
  File "XXXX/envs/pb-assembly-corrected/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "XXXX/envs/pb-assembly-corrected/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "XXXX/envs/pb-assembly-corrected/lib/python3.7/site-packages/falcon_kit/mains/ovlp_to_graph.py", line 1618, in <module>
    main(sys.argv)
  File "XXXX/envs/pb-assembly-corrected/lib/python3.7/site-packages/falcon_kit/mains/ovlp_to_graph.py", line 1614, in main
    ovlp_to_graph(args)
  File "XXXX/envs/pb-assembly-corrected/lib/python3.7/site-packages/falcon_kit/mains/ovlp_to_graph.py", line 1440, in ovlp_to_graph
    sg, sg_r, edge_data = generate_string_graph(args)
  File "XXXX/envs/pb-assembly-corrected/lib/python3.7/site-packages/falcon_kit/mains/ovlp_to_graph.py", line 898, in generate_string_graph
    nxsg.node[w]["best_in"] = v
AttributeError: 'DiGraph' object has no attribute 'node'
```
My understanding is that the attribute .node is [deprecated](https://github.com/networkx/networkx/commit/6b1ce03f485076d39994e8d624bbf6ca82466eb9) in networkx, and .nodes should be used instead. The last version of networkx to include both notations is 2.2. Both attributes yield exactly the same data, thus a clean fix would be to replace .node[] with .nodes[] in ovlp_to_graph.py:
```
898c898
<                 nxsg.nodes[w]["best_in"] = v
---
>                 nxsg.node[w]["best_in"] = v
1325c1325
<                     best_in_node = sg.nodes[t]["best_in"]
---
>                     best_in_node = sg.node[t]["best_in"]
```

An alternative would be to fix the networkx version to 2.2 in the bioconda recipe for pb-falcon.

**Comment by pb-cdunn** [comment]
Try

    conda update --all

When I run `conda list`, I have:
```
pb-falcon                 2.2.1            py37hed50d52_0    bioconda
```
```
% python3 -c 'import falcon_kit as m; print(m)'
falcon-kit 1.8.0 (pip thinks "falcon-kit 1.8.0")
<module 'falcon_kit' from '/home/UNIXHOME/cdunn/.conda/envs/pb-3.7/lib/python3.7/site-packages/falcon_kit/__init__.py'>

% grep best_in_node /home/UNIXHOME/cdunn/.conda/envs/pb-3.7/lib/python3.7/site-packages/falcon_kit/mains/ovlp_to_graph.py
                    best_in_node = sg.nodes[t]["best_in"]
```

We have seen a weird problem lately in Bioconda where `conda update --all` does not actually update **pb-falcon**. We have no idea why not. But you can `conda install pb-falcon==2.2.1`

**Comment by pb-cdunn** [comment]
(Unfortunately, we did not release a fix yet for incompatibility in falcon_unzip, but you can run falcon without falcon_unzip. We will release the unzip fix today.)

**Comment by ibebio** [comment]
Thanks for cheking. The pb-falcon update problem in pb-assembly was could have been caused by my [pull request](https://github.com/bioconda/bioconda-recipes/pull/18503) and is resolved now. My apologies for that! The pipeline crashed with the error that fc_ovl_filter did not have --min-idt and --ignore-indels options when we tested it with HiFi reads. I resorted to using version 0.3.4 as a fix, which was not necessary after all.
